### PR TITLE
Improve API documentation

### PR DESCRIPTION
**Pull request by bikamani**

### CODE REVIEW

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java [hunk 1]
```diff
@@ -722,6 +722,11 @@ public JobProperties exportDevices(String exportBlobContainerUri, Boolean exclud
      * Create a bulk export job.
      *
      * @param exportDevicesParameters A JobProperties object containing input parameters for export Devices job
+     *                                This API also supports identity bases storage authentication, identity authentication
```
Comment (abhipsaMisra) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java]: type: This API also supports identity ~bases~ based storage authenticatio

Comment (bikamani) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java]: done

**Pull request merged**